main.go

```diff
@@ -96,12 +95,19 @@ func main() {
 	dhis2Exporter := dhis2.NewExporter(dhis2Clients)
 	prometheus.MustRegister(dhis2Exporter)
 
-	// Register SendGrid exporters
+	// Register SendGrid exporters with time zones
 	apiKeys := make(map[string]string)
+	timeZones := make(map[string]*time.Location) // Added timeZones map
 	for _, account := range config.SendGridAccounts {
 		apiKeys[account.AccountName] = account.APIKey
+		loc, err := time.LoadLocation(account.TimeZone)
+		if err != nil {
+			log.Printf("Error loading time zone for account %s: %v", account.AccountName, err)
+			loc = time.UTC // Default to UTC if time zone cannot be loaded
+		}
+		timeZones[account.AccountName] = loc
 	}
-	sendgridExporter := sendgrid.NewExporter(apiKeys)
+	sendgridExporter := sendgrid.NewExporter(apiKeys, timeZones) // Passed timeZones to Exporter
```

Comment:
**suggestion**: I am thinking that a better interface would be to take multiple structs of configs as opposed to multiple maps where the ordering of the maps needs to be consistent.

confer: [Data Clumps](https://luzkan.github.io/smells/data-clump)